tests/unit/Core/Content/Cms/DataResolver/CmsSlotsMultiDataResolverTest.php

```diff
@@ -101,6 +104,9 @@ private function createResolver(string $type, string $definition): MultiCmsEleme
         return new MultiCmsElementResolver($type, $definition);
     }
 
+    /**
+     * @return SalesChannelRepository<covariant EntityCollection<covariant Entity>>|Stub
```

Comment:
```suggestion
     * @return SalesChannelRepository<covariant EntityCollection<covariant Entity>>&Stub
```
should be an intersection type and not a union, right?